src/vcpkg/base/system.process.cpp

```diff
@@ -322,6 +322,9 @@ namespace vcpkg
             L"VULKAN_SDK",
             // Enable targeted Android NDK
             L"ANDROID_NDK_HOME",
+            // Environmental variables generated automatically by Intel oneAPI after installation
+            L"ONEAPI_ROOT",
+            L"IFORT_COMPILER19",
```

Comment:
```suggestion
            L"IFORT_COMPILER19",
            L"IFORT_COMPILER20",
            L"IFORT_COMPILER21",
```

maybe move all those variables into the triplet via `VCPKG_ENV_PASSTHROUGH_UNTRACKED` ? 
And I don't mean only the ones I added I mean all of them.